src/models/backup.ts

```diff
@@ -83,7 +83,9 @@ class Drive {
                     const res = JSON.parse(xhr.responseText);
                     if (res.error) {
                       if (res.error.code === 401) {
-                        localStorage.driveToken = '';
+                        if (navigator.userAgent.indexOf('Chrome') === -1) {
```

Comment:
Any detail about the exception for Chrome? Why should we not remove cache in Chrome?

Reply:
It's so it can remove the token from the identity cache, and after that remove the token from storage. Otherwise it will just keep returning the revoked token.
https://github.com/Authenticator-Extension/Authenticator/blob/777d47a2751336948b957a84f5ccf975ca786103/src/models/backup.ts#L111-L114

Reply:
Then why should we skip clearing cache in Chrome

Reply:
You can't remove a token from the cache without knowing what it is. After the chrome cached token is removed, the token is removed from localStorage like normal.